src/prefect/settings.py

```diff
@@ -534,6 +590,113 @@ class APISettings(PrefectBaseSettings):
     )
 
 
+class CLISettings(PrefectBaseSettings):
+    """
+    Settings for controlling CLI behavior
+    """
+
+    model_config = SettingsConfigDict(
+        env_prefix="PREFECT_CLI_", env_file=".env", extra="ignore"
+    )
+
+    colors: bool = Field(
+        default=True,
+        description="If True, use colors in CLI output. If `False`, output will not include colors codes.",
+    )
+
+    prompt: Optional[bool] = Field(
+        default=None,
+        description="If `True`, use interactive prompts in CLI commands. If `False`, no interactive prompts will be used. If `None`, the value will be dynamically determined based on the presence of an interactive-enabled terminal.",
+    )
+
+    wrap_lines: bool = Field(
+        default=True,
+        description="If `True`, wrap text by inserting new lines in long lines in CLI output. If `False`, output will not be wrapped.",
+    )
+
+
+class ClientMetricsSettings(PrefectBaseSettings):
+    """
+    Settings for controlling metrics reporting from the client
+    """
+
+    model_config = SettingsConfigDict(
+        env_prefix="PREFECT_CLIENT_METRICS_", env_file=".env", extra="ignore"
+    )
+
+    enabled: bool = Field(
+        default=False,
+        description="Whether or not to enable Prometheus metrics in the client.",
+        # Using alias for backwards compatibility. Need to duplicate the prefix because
+        # Pydantic does not allow the alias to be prefixed with the env_prefix.
+        validation_alias=AliasChoices(
+            "prefect_client_metrics_enabled", "prefect_client_enable_metrics"
+        ),
+    )
```

Comment:
This is the field that was moved from `PREFECT_CLIENT_ENABLE_METRICS` to `PREFECT_CLIENT_METRICS_ENABLED`.